pkg/sfu/downtrack.go

```diff
@@ -566,7 +567,18 @@ func (d *DownTrack) CloseWithFlush(flush bool) {
 	// display buffer and there could be a brief moment where the previous stream is displayed.
 	d.logger.Infow("close down track", "peerID", d.peerID, "trackID", d.id, "flushBlankFrame", flush)
 	if flush {
-		_ = d.writeBlankFrameRTP()
+		doneFlushing := make(chan struct{})
+		go func() {
```

Comment:
A bit weary of this. Will this goroutine run quickly enough? What if the renegotiate happens and the SSRC is flushed from client's view of streams before this happens and the blank frames are rendered useless?